service/templates/projects.html

```diff
@@ -642,15 +642,15 @@ <h3>Commit Queue</h3>
           <!-- COMMIT QUEUE PATCH DEFINITIONS -->
           <div ng-show="settingsFormData.commit_queue.enabled">
             <div>
-              <div>
-                <label for="mergeMethodSelect"> Merge Method: </label>
-                <select id="mergeMethodSelect" ng-model="settingsFormData.commit_queue.merge_method" ng-options="option for option in validMergeMethods"></select>
-              </div>
               <div>
                 <label for="PatchTypeSelect"> Patch Type: </label>
                 <select id="PatchTypeSelect" ng-model="settingsFormData.commit_queue.patch_type" ng-options="option for option in validPatchTypes"></select>
                 <div class="muted small">Patch Type is PR for merging GitHub PRs or CLI for merging patches submitted through Evergreen's CLI.</div>
               </div>
+              <div>
+                <label for="mergeMethodSelect"> Merge Method: </label>
+                <select id="mergeMethodSelect" ng-disabled="settingsFormData.commit_queue.patch_type!=='PR'" ng-model="settingsFormData.commit_queue.merge_method" ng-options="option for option in validMergeMethods"></select>
```

Comment:
maybe we should also pin `merge_method` to rebase in this case?

Reply:
I'd prefer to not do that. If you change the merge method on accident, I think it's better to keep the old selection if you want to change it back. I think disabling the control is clear enough